src/main/java/me/ajaja/global/common/BaseTime.java

```diff
@@ -75,6 +76,10 @@ public boolean isWithinAMonth(BaseTime time) {
 		return zonedDateTime.isBefore(time.oneMonthLater()) && zonedDateTime.isAfter(time.zonedDateTime);
 	}
 
+	public long getBetweenDays(BaseTime endDate) {
+		return ChronoUnit.DAYS.between(instant, BaseTime.now().getInstant());
+	}
+
```

Comment:
get이라는 네이밍은 자바 빈즈 규약에 따라 내부의 값을 꺼내는 용도에 가깝습니다. 
기존에 Import 되어있는 것을 활용하고 네이밍을 살짝 손봐서 다음과 같은 코드는 어떤가요?

```suggestion
	public long betweenDays(BaseTime compare) {
		return Duration.between(this.instant, compare.instant).toDays();
	}
```

Reply:
68d303f1cd0009f4955b7e21d55f734ea2c07101
수정 완료하였습니다!